Library/Homebrew/cask/spec/cask/artifact/uninstall_spec.rb

```diff
@@ -7,12 +7,28 @@
     Hbc::Artifact::Uninstall.new(cask, command: Hbc::FakeSystemCommand)
   }
 
+  let(:absolute_path) { Pathname.new("#{TEST_TMPDIR}/absolute_path") }
+  let(:path_with_tilde) { Pathname.new("#{TEST_TMPDIR}/path_with_tilde") }
+  let(:glob_path1) { Pathname.new("#{TEST_TMPDIR}/glob_path1") }
+  let(:glob_path2) { Pathname.new("#{TEST_TMPDIR}/glob_path2") }
+  let(:directory_path) { Pathname.new("#{TEST_TMPDIR}/directory_path") }
+
   before(:each) do
+    FileUtils.touch(absolute_path)
+    FileUtils.touch(path_with_tilde)
+    FileUtils.touch(glob_path1)
+    FileUtils.touch(glob_path2)
+    directory_path.mkdir
+    ENV["HOME"] = TEST_TMPDIR
     shutup do
       InstallHelper.install_without_artifacts(cask)
     end
   end
 
+  after(:each) do
+    directory_path.rmdir
```

Comment:
The `directory_path` should go into the `context "when using rmdir" do` block below. Also, rename this to `empty_directory_path`.